contrib/colorschemes/monochrome

```diff
@@ -0,0 +1,11 @@
+color listnormal         default default dim
+color listnormal_unread  default default 
```

Comment:
I expect the "code formatting" check is is failing due to a bit of trailing whitespace at the end of this line